src/App.css

```diff
@@ -9,7 +9,13 @@
 	position: relative;
 	height: 22vmin;
 	width:300px;
-	margin:auto;
+  margin:auto;
+  background: url(Background.png);
+  background-repeat: no-repeat;
+  background-size: 300px;
+  background-position-y: -5px ;
```

Comment:
Nice birds, but why do we need background-position-y? Couldn't we just crop the image?